Vagrantfile

```diff
@@ -20,26 +19,7 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
 
     # VM network config.
     config.vm.network "forwarded_port", guest: 80, host: 8080
-    #config.vm.network "forwarded_port", guest: 443, host: 44343
-    config.vm.synced_folder ".", "/vagrant", :create => true
-    config.ssh.forward_agent = true
+    config.vm.network "private_network", ip: "192.168.50.4"
```

Comment:
Because a specific IP address allows the same thing to be added to an Ansible hosts file and then ansible commands can be run on the vagrant host directly, without having to run through the vagrant provisioning process.